File: Praktikum I Modul 5/Modul 5/Python/PRAK501-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# fungsi
def MaxBilangan(a, b, c, d):
    # pengkondisian
    if (a > b and a > c and a > d): return a
    elif (b > c and b > d): return b 
    elif (c > d): return c
    else: return d

# input 
a, b, c, d = map(int, input().split())

hasil = MaxBilangan(a, b, c, d)
# output
print(hasil)

File: Praktikum I Modul 5/Modul 5/C/PRAK501-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

// fungsi 
int MaxBilangan(int a, int b, int c, int d){
    // pengkondisian
    if (a > b && a > c && a > d){
        return a;
    } else if (b > c && b > d){
        return b;
    } else if (c > d){
        return c;
    } else {
        return d;
    }
}

int main(){
    // deklarasi variabel
    int a, b, c, d;
    int hasil;

    // input
    scanf("%d %d %d %d", &a, &b, &c, &d);

    hasil = MaxBilangan(a, b, c, d);
    // output
    printf("%d", hasil);

    return 0;
}